backend/config.js

```diff
@@ -34,12 +34,12 @@ module.exports = {
       url: process.env.OAUTH_ENDPOINT_URL || 'http://localhost:9090/auth/',
       ui_url: process.env.OAUTH_ENDPOINT_UI_URL || process.env.OAUTH_ENDPOINT_URL || 'http://localhost:9090/auth/',
       timeoutSeconds: process.env.API_ENDPOINT_TIMEOUT_SECONDS || 10,
-      clientId: process.env.API_CLIENT_ID || 'elite2apiclient',
+      clientId: process.env.API_CLIENT_ID || 'prisonapiclient',
```

Comment:
Does this secret work with auth and elite running locally?

Reply:
when I fired-up the services using docker-compose and ran prisonstaffhub from dev I was able to see the search page so that tells me yes was working